CONTRIBUTING.md

```diff
@@ -28,97 +29,94 @@ The commands are providing as npm-critpts, please executing `npm run {{ Command
 
 ### Clone and create branch
 
-#### 1.Please clone from GitHub repository at [here](https://github.com/kintone-labs/kintone-ui-component) to local
+#### 1. Clone from GitHub repository [here](https://github.com/kintone-labs/kintone-ui-component) to local
 
 ```sh
 git clone git@github.com:kintone-labs/kintone-ui-component.git
 ```
 
-#### 2.Check out to branch for dev
-Please refer [Branch naming rule](#Branch) and create a new branch for developing.
+#### 2. Checkout to branch for development
+Please refer to [Branch naming rule](#Branch) and create a new branch for development from the next version branch.
 ```sh
 git branch <branch name>
 git checkout <branch name>
 ```
 ### Setup
-This repository is a Siggle-repo Monolith using npm.
+This repository is a Singgle-repo using npm.
 ```sh
 cd kintone-ui-component
 npm ci
 ```
 ### Develop
 #### 1. Implement your functions/features
-#### 2  Eslint check
+#### 2. ESlint check
 ```sh
 npm run es-lint
 ```
 #### 3. Start storybook to confirm your functions/features
 ```sh
 npm run storybook
 ```
-#### 4. Commit and Create Pull-Request
-Please refer [Pull Request naming rule](#Pull-Request)  
+#### 4. Commit and Create Pull Request
+Please refer to [Pull Request naming rule](#Pull-Request).  
 
 ### Build
 ```sh
+npm run build:umd_dev
+or
```

Comment:
Why did you add only umd_dev not including esm??
My suggestion was below. Please check it.
https://github.com/kintone-labs/kintone-ui-component/pull/898#discussion_r796604907